source/configure-connection.txt

```diff
@@ -0,0 +1,175 @@
+  the ``parse_uri()`` method
+
+.. tip::
+
+   To learn how to install the Django ODM and create a
+   Django project, visit the :ref:`django-get-started` tutorial.
+
+.. _django-connection-configure-manual:
+
+Manually Configure Database Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To manually configure your connection to MongoDB, update
+the ``DATABASES`` variable in your project's ``settings.py``
+file. Set the ``DATABASES`` variable to a dictionary value containing
+the database settings. You must configure the dictionary's
+``default`` key to set MongoDB as the default database
+connection.
+
+To configure the ``default`` key, assign a nested dictionary as its value.
+This nested dictionary has the following keys:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Key
+     - Description
+
+   * - **ENGINE**
+     - The backend driver to use for the connection. Set this key to ``"django_mongodb_backend"``.
+
+   * - **HOST**
+     - | Your connection URI. For localhost connections, this key is optional. 
+       | For SRV connections, you must include a scheme prefix (``mongodb+srv://``).
+       |
+       | If connecting to a replica set or sharded cluster with multiple hosts, specify 
+       | each host separated by a comma.
+
+   * - **NAME**
+     - The database you want to use.
+
+   * - **USER**
+     - The username for authenticating to the database.
+
+   * - **PASSWORD**
+     - The password for your database user.
+
+   * - **PORT**
+     - | The port number on which the database server is listening. 
+       | For MongoDB Atlas connections, this key is optional.
+
+   * - **OPTIONS**
+     - A dictionary of additional connection options for the database. This key is optional.
+
+.. _django-manual-config-example:
+
+Example 
+```````
+
+This example specifies the ``DATABASES`` variable to connect
+to a MongoDB deployment with the following configuration:
+
+- Connects to the ``my_database`` database
+- Provides authentication information for a database user
+  whose username is ``my_user`` and password is ``my_password``
+- Uses the default MongoDB port (``27017``)
+- Sets the ``retryWrites`` connection option to ``true``,
+  which configures the driver to automatically retry certain
+  write operations if they fail
+- Sets the ``w`` connection option to ``majority``,
+  which configures the driver to wait for acknowledgement from a majority
+  of replica set members before performing write operations
+
+.. code-block:: python
+
+    DATABASES = {
+        "default": {
+            "ENGINE": "django_mongodb_backend",
+            "HOST": "mongodb+srv://cluster0.example.mongodb.net",
+            "NAME": "my_database",
+            "USER": "my_user",
+            "PASSWORD": "my_password",
+            "PORT": 27017,
+            "OPTIONS": {
+                "retryWrites": "true",
+                "w": "majority",
+            },
+        },
+    }
+
+.. tip::
```

Comment:
s: move this to the OPTIONS row above (and take out of tip box to avoid nested admonition)